### DISCUSSION TITLE
Windows 7

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#2308] [General]
I want to use ungoogled-chromium on Windows 7.
If possible, I would like to see support for Windows 7.

EDIT: I gave up. Thank you.

**Comment by PF4Public** [comment]
Have you ever considered starting using Linux? It ain't a silver bullet of-course, it has its own quirks for sure.

This allows one to have atleast an up-to-date system. Windows 7 is aging inevitably. I know what I'm talking about since I'm still using Windows XP and Windows 7 on my daily basis.

Please also see https://github.com/ungoogled-software/ungoogled-chromium/issues/2187, https://github.com/ungoogled-software/ungoogled-chromium/issues/2206 and https://github.com/ungoogled-software/ungoogled-chromium-windows/issues/193

**Comment by llExSt** [comment]
You can check out [Supermium](https://github.com/win32ss/supermium)
It ain't ungoogled-chromium but if you still want to continue using an updated chrome on W7 there's at least an alternative.

Found another that uses some ungoogled-chromium [patches](https://github.com/Alex313031/thorium/issues/115#issuecomment-1454352160), [Thorium](https://github.com/Alex313031/thorium-win7).
